inc/saimacsec.h

```diff
@@ -535,23 +558,33 @@ typedef enum _sai_macsec_sc_attr_t
      * @brief End of MACsec Secure Channel attributes
      */
     SAI_MACSEC_SC_ATTR_END,
+
+    /**
+     * @brief Custom range base value
+     */
+    SAI_MACSEC_SC_ATTR_CUSTOM_RANGE_START = 0x10000000,
+
+    /**
+     * @brief End of custom range base
+     */
+    SAI_MACSEC_SC_ATTR_CUSTOM_RANGE_END
 } sai_macsec_sc_attr_t;
 
 /**
- * @brief MACsec Secure Channel counter IDs in sai_get_macsec_sc_stats_ext() call
+ * @brief MACsec Secure Channel counter IDs in sai_get_macsec_sc_stats() call
  */
 typedef enum _sai_macsec_sc_stat_t
 {
     /**
      * @brief Control packets which are not secured (using MACsec uncontrolled port) -
      * not in 802.1ae MIB
      */
-    SAI_MACSEC_SC_STAT_CONTROL_PACKETS,
+    SAI_MACSEC_SC_STAT_CONTROL_PKTS,
 
     /**
      * @brief IEEE 802.1ae defined OutPktsUntagged or InPktsUntagged
      */
-    SAI_MACSEC_SC_STAT_PACKETS_UNTAGGED,
+    SAI_MACSEC_SC_STAT_PKTS_UNTAGGED,
```

Comment:
what was wrong with long "packets" prefixes ?

Reply:
Tried to closely match the MIB object names in 802.1ae spec (InPktsUntagged and OutPktsUntagged).